### PR TITLE
feat/init skeleton

### PR DESCRIPTION
**Pull request by anhthichieu**

### CODE REVIEW

**Comment by anhthichieu** [comment]
@budtr Please spend time looking through this PR when you get a chance. Thanks.

File: src/views/Workflows.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+          {
+            'trigger-content': () => h(Icon, { icon: 'bi:trash' }),
+          }
+        ),
+      ]);
+    },
+  },
+]);
+
+const createData = () => [
```
Comment (budtr) [inline src/views/Workflows.vue:171]: @anhthichieu can you please move it to mock file and import it from store/api services?

File: src/views/History.vue [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  },
+  {
+    label: 'Failed',
+    value: 'failed',
+    icon: 'ic:round-cancel',
+    iconColor: '#D03050FF',
+  },
+]);
+
+function getChainImg(chain) {
```
Comment (budtr) [inline src/views/History.vue:45]: @anhthichieu this should be in a config file

Comment (anhthichieu) [reply src/views/History.vue]: @budtr fixed

**Pull request merged**